src/resources/js/admin/tickets-emails.js

```diff
@@ -46,6 +46,8 @@ tribe.tickets.emails = {};
 		formFooterContent: 'tec-tickets-emails-footer-content',
 		formFooterCredit: 'tec-tickets-emails-footer-credit',
 		formQrCodes: 'tec-tickets-emails-ticket-include-qr-codes',
+		formRsvpUsingTicketEmail: 'input[name=tec-tickets-emails-rsvp-use-ticket-email]',
+		formPreviewButton: 'button[name=tec-tickets-emails-preview]',
```

Comment:
Let's move the `button[name=` to where we get the value as we do with the other fields.